exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/EndToEndTest.java

```diff
@@ -17,27 +17,118 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.ServerSocket;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
 import java.util.concurrent.TimeUnit;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+class TestSpanData implements SpanData {
```

Comment:
Sorry, why can't you just use [TestSpanData from the SDK](https://github.com/open-telemetry/opentelemetry-java/blob/c3a893a88ca97c4dcc01e1e0f289eed1ffd3b549/testing_internal/src/main/java/io/opentelemetry/sdk/trace/TestSpanData.java#L43) again?

It has been moved to a different package, but we should still be able to use it?

Reply:
@nilebox any help you can offer on this front is greatly welcomed. I tried pulling `TestSpanData` in from OpenTelemetry but I _believe_ they moved that into a [non-published package recently](https://github.com/open-telemetry/opentelemetry-java/pull/1512)

As a sanity check (after hunting manually through MVN etc.) Gradle solidly does not like `testing-internal` either:

```
Could not resolve **io.opentelemetry:testing-internal:0.8.0.**
```

If we're cool with using Gradle's source dependency for this, I can definitely make this work, but I'm also assuming I'm just solidly missing something obvious 😅

Reply:
Sorry I missed that they don't release a Maven artifact with this class anymore :(
I think I'd still prefer to have a TestSpanData having settable properties rather than hardcode ones, but that works as well.

Reply:
You may use `@AutoValue` to avoid too much boilerplate if that's your concern: https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/d86c5b1e69356b085ed91ea60fc2636590491877/exporters/trace/src/main/java/com/google/cloud/opentelemetry/trace/TraceConfiguration.java#L20-L22

Reply:
The easiest would be to just copy over https://github.com/open-telemetry/opentelemetry-java/blob/master/testing_internal/src/main/java/io/opentelemetry/sdk/trace/TestSpanData.java since they are also using AutoValue, and leave a NOTE comment with a link to the original, so that we may keep them in sync if there are any changes in the interface.